jwst/pipeline/calwebb_spec3.py

```diff
@@ -2,7 +2,7 @@
 
 from ..stpipe import Pipeline
 from .. import datamodels
-from ..exp_to_source import exp_to_source
+from ..exp_to_source import multislit_to_container
 
 # step imports
 from ..skymatch import skymatch_step

```

Comment:
Sorry, but yes, I was referring to the `--steps.?????` specification the user sees. So, it looks like it should be `--steps.mir_imatch` and no longer `--steps.sky_match`